wire-grpc-sample/client/build.gradle

```diff
@@ -30,10 +30,10 @@ wire {
 }
 
 dependencies {
-  implementation project(':protos')
-  implementation deps.kotlin.stdlib
-  implementation deps.kotlin.coroutines
-  implementation deps.kotlin.coroutinesAndroid
+  implementation project(':wire-grpc-sample:protos')
+  implementation deps.kotlin.coroutines.core.common
```

Comment:
Can we use `core` transitively through `android`? Also, probably just `core`, not `core.common`.

Reply:
fixed